apps/arena/lib/arena/game/player.ex

```diff
@@ -175,6 +176,13 @@ defmodule Arena.Game.Player do
         {auto_aim?, skill_direction} =
           skill_params.target
           |> Skill.maybe_auto_aim(skill, player, targetable_players(game_state.players))
+          |> case do
+            {false, _} ->
+              Skill.maybe_auto_aim(skill_params.target, skill, player, Crate.interactable_crates(game_state.crates))
```

Comment:
Why we need this case? 🤔 
I wonder if we can do something like `targetable_players(game_state.players) ++ Crate.interactable_crates(game_state.crates))`

Reply:
The thing is that you want to prioritize player over crates, this case match if there's no player that you can auto aim, so you try to auto aim a crate

Reply:
Okay, lets iterate but I think we can find a way to improve this an order the entities by some kind of priority